tests/kafkatest/services/kafka/kafka.py

```diff
@@ -983,7 +988,10 @@ def _topic_command_connect_setting(self, node, force_use_zk_connection):
         bootstrap server, otherwise returns zookeeper connection string.
         """
         if not force_use_zk_connection and self.all_nodes_topic_command_supports_bootstrap_server():
-            connection_setting = "--bootstrap-server %s" % (self.bootstrap_servers(self.security_protocol))
+            if self.interbroker_security_protocol == SecurityConfig.PLAINTEXT:
+                connection_setting = "--bootstrap-server %s" % (self.bootstrap_servers(self.interbroker_security_protocol))
+            else:
+                connection_setting = "--bootstrap-server %s" % (self.bootstrap_servers(self.security_protocol))
```

Comment:
Is this assuming that `else` section always means PLAINTEXT since there are no security configs here?

Reply:
This code is identifying the port to contact when we are using `--bootstrap-server` instead of `--zookeeper`, and we use the port associated with `PLAINTEXT` rather than `SASL_{PLAINTEXT,SSL}` or `SSL` if `PLAINTEXT` is in use.  The assumption is that if `PLAINTEXT` is in use it will be the inter-broker security protocol rather than (or in addition to) the client security protocol, which is the case.